soroban-sdk/src/token.rs

```diff
@@ -27,11 +27,67 @@ pub use TokenInterface as Interface;
 pub use TokenClient as Client;
 
 /// Interface for Token contracts, such as the Stellar Asset Contract.
+///
+/// Defined by [SEP-41].
+///
+/// [SEP-41]: https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0041.md
+///
+/// The token interface provides the following functionality.
+///
+/// If a contract implementing the interface does not support some of the
+/// functionality, it should return an error.
+///
+/// The interface does not define any set of standard errors. Errors can be
+/// defined by the implementing contract.
+///
+/// ## Meta
+///
+/// Tokens implementing the interface expose meta functions about the token:
+/// - [`decimals`][Self::decimals]
+/// - [`name`][Self::name]
+/// - [`symbol`][Self::symbol]
+///
+/// ## Balances
+///
+/// Tokens track a balance for each address that holds the token. Tokens implementing the interface expose
+/// a single function for getting the balance that an address holds:
+/// - [`balance`][Self::balance]
+///
+/// ## Transfers
+///
+/// Tokens allow holders of the token to transfer tokens to other addresses.
+/// Tokens implementing the interface expose a single function for doing so:
+/// - [`transfer`][Self::transfer]
+///
+/// ## Burning
+///
+/// Tokens allow holders of the token to transfer tokens to other addresses.
```

Comment:
Looks like this is the `Transfers` description.

Reply:
Thanks, fixed in be0035c.